css/style.css

```diff
@@ -220,14 +225,18 @@ main {
   border-radius: 2px;
 }
 
-.toggle-inner {
+input[type="checkbox"] {
+  display: none;
```

Comment:
Also a few properties is way simpler than the tonne of JS to handle keyboard tab, space and clicking to change the class on a div.